library/std/src/fs.rs

```diff
@@ -1365,6 +1365,30 @@ impl FileTimes {
 impl Permissions {
     /// Returns `true` if these permissions describe a readonly (unwritable) file.
     ///
+    /// Warning: This function does not take ACLs or Unix group membership into
+    /// account.
+    ///
+    /// ## Windows
+    ///
+    /// On Windows this returns [`FILE_ATTRIBUTE_READONLY`](https://docs.microsoft.com/en-us/windows/win32/fileio/file-attribute-constants).
+    /// If `FILE_ATTRIBUTE_READONLY` is set then writes to the file will fail
+    /// but the user may still have permission to change this flag.
+    /// The behaviour of this attribute for directories depends on the Windows
```

Comment:
In the doc we prefer American spelling:

```suggestion
    /// The behavior of this attribute for directories depends on the Windows
```